content/overview/pricing/free-limited-sandbox.md

```diff
@@ -8,44 +8,42 @@ aliases:
+Then, if you’re interested in [purchasing full access]({{< relref "overview/pricing/rates.md" >}}) (whether for **Prototyping** or for production systems at the **Open Data**, **FISMA Low**, or **FISMA Moderate** levels), fill out [this interest form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSevZfuJ_4KE-MZlm9gttYfsXQp0PJL7OR6k6LbZ9XnFn-oA6g/viewform) or email us again.
 
-## To keep in mind before you try your sandbox
+## Keep in mind before you try your sandbox
 
 * If your agency has not already integrated its single-sign on authentication provider with cloud.gov (only GSA, EPA, and FDIC have done this so far), you will access your sandbox through a [cloud.gov account in the GovCloud environment]({{< relref "docs/getting-started/accounts.md#cloud-gov-accounts" >}}). This account requires multi-factor authentication using a mobile app such as [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en) or [Authy](https://www.authy.com/app/mobile/). If you cannot install or use these apps, such as if your workplace prohibits mobile phones or mobile phone cameras, you might not be able to set up access. (Paid access packages support integration with your agency single sign-on authentication provider.)
-* If your organization prohibits installing the [cloud.gov command line interface]({{< relref "docs/getting-started/setup.md#set-up-the-command-line" >}}) on your computer, you won't be able to deploy applications in your sandbox. (For paid access packages, we can coordinate with your agency to help them approve this tool.)
+* If your organization prohibits installing the [cloud.gov command line interface]({{< relref "docs/getting-started/setup.md#set-up-the-command-line" >}}) on your computer, you won’t be able to deploy applications in your sandbox. (For paid access packages, we can coordinate with your agency to help them approve this tool.)
 
 ## A few things you can try in your sandbox
 
-On a technical level, a sandbox is a specially-limited ["space"]({{< relref "docs/getting-started/concepts.md#spaces" >}}) within a sandbox-only ["organization"]({{< relref "docs/getting-started/concepts.md#organizations" >}}) that is managed by cloud.gov.
+On a technical level, a sandbox is a specially-limited ["space"]({{< relref "docs/getting-started/concepts.md#spaces" >}}) within a sandbox-only ["organization"]({{< relref "docs/getting-started/concepts.md#organizations" >}}) that is managed by cloud.gov. In paid accounts, you would manage your own organization and set up spaces for different applications. With a sandbox, you are managing one space and the applications and services within it.
```

Comment:
A person can have both access to a free sandbox space and a paid org, but their access to a paid org might not be "Org Manager" access - they might just have "Space Developer" access to spaces there. Also often an org has spaces for same or similar versions of the same application (such as a development and a production space) rather than different applications.